extensions/wikia/Email/EmailController.class.php

```diff
@@ -26,6 +26,98 @@
 	// Used when needing to specify an anonymous user from the external API
 	const ANONYMOUS_USER_ID = -1;
 
+    const SOCIALS = [
+        'en' => [
+            'Facebook',
+            'Twitter',
+            'YouTube',
+            'Instagram'
+        ],
+        'zh-hant' => [
+            'Facebook'
+        ],
+        'zh-hans' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "zh-hans" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)